bash_completion

```diff
@@ -3,19 +3,19 @@
 # bash completion for Node Version Manager (NVM)
 
 if ! command -v nvm &> /dev/null; then
-    return
+  return
 fi
 
 __nvm_generate_completion()
 {
```

Comment:
these curly braces should all be in OTBS, not in allman style.

Reply:
Sure